client/blocks/login/two-factor-authentication/push-notification-illustration.jsx

```diff
@@ -10,19 +11,28 @@ const COLOR_BLUE_10 = PALETTE[ 'WordPress Blue 10' ];
 const COLOR_BLUE_20 = PALETTE[ 'WordPress Blue 20' ];
 const COLOR_BLUE_40 = PALETTE[ 'WordPress Blue 40' ];
 const COLOR_BLUE_50 = PALETTE[ 'WordPress Blue 50' ];
+
+const COLOR_GREEN_40 = PALETTE[ 'Jetpack Green 40' ];
+
+const COLOR_GRAY_5 = PALETTE[ 'Gray 5' ];
+const COLOR_GRAY_20 = PALETTE[ 'Gray 20' ];
+const COLOR_GRAY_80 = PALETTE[ 'Gray 80' ];
+
 const COLOR_WHITE = PALETTE[ 'White' ]; // eslint-disable-line dot-notation
 
+const displayJetpackAppBranding = config.isEnabled( 'jetpack/app-branding' );
+
```

Comment:
@SiobhyB In theory, we can deploy these changes without a feature flag, the only requirement is to hold off the deployment until we update the 2FA push notification logic in WPCOM. Therefore, either we create another feature flag only for this change or simply hold off the merge.

Reply:
Thanks for the clarification! I've gone ahead to remove this flag in https://github.com/Automattic/wp-calypso/pull/70832/commits/5a9ebfb06eaca32bb72f2d259526c1cb33011c61.